Rakefile

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/rake
+require 'yaml'
+require 'json'
+require 'net/http'
+require 'uri'
+require 'plist'
+
+desc 'Bootstrap this repository for development'
+task :bootstrap do
+  [:StencilSwiftKit, :SwiftGenKit, :SwiftGen, :templates].each do |repository|
+    next if Dir.exists? "#{repository}"
+
+    sh "git clone git@github.com:SwiftGen/#{repository}.git --recursive"
+    Dir.chdir("#{repository}") do
+      sh "git submodule update --init --recursive"
+    end
+  end
+end
```

Comment:
Not sure if this would be the same task for both or separate ones, but what about having a task which update all submodules of each subrepo at once (e.g. if we add commits to the templates repo and then want that all repos align their templates submodule to those last commit)?

We could also have tasks to:
* `git pull` all repos to make sure we get all commits added to the remote
* `git rev-parse --abbrev-ref HEAD` on all repos, to show an overview of all repos and on which branch each repo currently is locally and `git status` to show if it has a dirty working copy or not

(Just ideas, not necessary for this PR)